configure.ac

```diff
@@ -370,6 +370,11 @@ case "$ac_cv_search_crypt" in
 	-l*) LIBCRYPT="$ac_cv_search_crypt" ;;
 	*) LIBCRYPT="" ;;
 esac
+AC_SEARCH_LIBS([crypt_r],[$crypt_libs])
+case "$ac_cv_search_crypt_r" in
+	-l*) LIBCRYPT="$LIBCRYPT $ac_cv_search_crypt_r" ;;
+	*) LIBCRYPT="$LIBCRYPT" ;;
```

Comment:
What `LIBCRYPT="$LIBCRYPT"` is expected to do?

Reply:
Nothing, I'll drop it.